### DISCUSSION TITLE
Panoptic segmentation (Grounded Segment Anything + CLIPSeg)

### DISCUSSION THREAD

**Discussion by ghost** [IDEA-Research/Grounded-Segment-Anything#128] [Show and tell]
Thanks for the awesome work on both Grounding DINO and Grounded SAM!
I built a new pipeline for zero-shot segmentation on top of your work:
[https://github.com/segments-ai/panoptic-segment-anything](https://github.com/segments-ai/panoptic-segment-anything)

**Comment by rentainhe** [comment]
Really really awesome work! We will highlight it in our project!
